tools/generator/game.py

```diff
@@ -171,6 +173,56 @@ def check_field_unicity(self):
                         .format(origin_type, origin, name, def_type, def_name))
                 used_field_names[name] = (origin_type, origin)
 
+    def check_reserved_keywords(self):
+        '''Check that no identifier is in conflict with a reserved keyword'''
```

Comment:
Seems like you could easily factor this with a nested function:

```
def check_ident(name, what):
    if conflicts_with_reserved_keyword(name):
        raise GameError("{} '{}': ...".format(what, name))
```

Reply:
done